src/transformers/models/dac/configuration_dac.py

```diff
@@ -0,0 +1,49 @@
+"""Dac model configuration"""
+
+import numpy as np
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+class DacConfig(PretrainedConfig):
+    r"""
+    """
+
+    model_type = "dac"
+
+    def __init__(
+        self,
+        encoder_dim=64,
+        encoder_rates=[2, 4, 8, 8],
+        latent_dim=None,
+        decoder_dim=1536,
+        decoder_rates=[8, 8, 4, 2],
+        n_codebooks=9,
+        codebook_size=1024,
+        codebook_dim=8,
+        quantizer_dropout=False,
```

Comment:
Should this be a bool? Shouldn't it be a dropout factor?

Reply:
In the original repository it is defined as a `bool` (see [here](https://github.com/descriptinc/descript-audio-codec/blob/c7cfc5d2647e26471dc394f95846a0830e7bec34/dac/model/dac.py#L158)) and set to `False` in the checkpoints metadata. 

It does makes sense to replace it with a float between 0 and 1, and set the default to 0 tough.